op.c

```diff
@@ -12537,6 +12537,14 @@ Perl_newSTUB(pTHX_ GV *gv, bool fake)
     return cv;
 }
 
+/*
+=for apidoc newFORM
+
+Constructs, checks, and returns a format op.
```

Comment:
Well newFORM() is void, so it doesn't return anything, let alone an op. It actually adds a 'form' CV to the FORMAT slot of the STDOUT / format name GV